internal/mongodoc/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package mongodoc_test // import "gopkg.in/juju/charmstore.v5-unstable/internal/mongodoc"
```

Comment:
While you are correct, note that each existing file in this code base duplicates the specifier.  Given that the precedent is mildly inconsistent (only 3 files don't do it), I don't mind dropping the specifier.